UnanimusProject/res/layout/group_settings_picker_activity.xml

```diff
@@ -1,121 +1,163 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-    style="@style/MatchParent"
-    android:fillViewport="false">
-
-    <LinearLayout xmlns:tools="http://schemas.android.com/tools"
-        android:id="@+id/group_settings_picker_activity"
-        style="@style/RootLayout">
-
-        <include layout="@layout/activity_title" />
+    style="@style/RootLayout"
```

Comment:
Only one element on the page should have "@style/RootLayout" or else the margins will be wrong